portfolio/src/main/webapp/comment.js

```diff
@@ -85,4 +123,53 @@ function clearComments(space) {
   while (space.firstChild) {
     space.removeChild(space.firstChild);
   }
+}
+
+function fetchLogin() {
+  fetch('/login')
+  .then(response => response.json()) // Convert to json
+  .then((status) => {
+    // Logged in 
+    if (status.login === 'true' && status.email != 'test@example.com') {
+      document.getElementById('comment-form').classList.remove("invisible");
```

Comment:
What would happen if I manually edited the html to remove this CSS class? Would it allow me to submit comments without being logged in?

Reply:
Discussed in 1:1